dfvfs/lib/gzipfile.py

```diff
@@ -152,41 +152,71 @@ def __init__(
     self.operating_system = None
     self.original_filename = None
 
+    # Read the member data to determine the uncompressed data size and
```

Comment:
What does this comment apply to? The next call is just getting the size, and the read on line 160 doesn't read the whole member data, just the header. Did you mean "member header" not "member data"?

Please review this comment and clarify.

Reply:
I'll move it.